docs/modules/ROOT/pages/explanation/storage-classes.adoc

```diff
@@ -33,6 +33,8 @@ See the xref:how-to/encrypted-volumes.adoc[Setup Encrypted Volumes] guide.
 
 You may want to consider using the `bulk` storage class, if you're working with very large amounts of data but read/write performance isn't critical.
 
+Please be aware that you can only provision `bulk` volumes in steps of 100GB, since cloudscale.ch doesn't provide more fine-grained volume sizes for `bulk` storage.
+
```

Comment:
```suggestion
[NOTE]
.Special case cloudscale.ch `bulk`
====
Please be aware that you can only provision `bulk` volumes in steps of 100 GB, since cloudscale.ch doesn't provide more fine-grained volume sizes for `bulk` storage.
====
```

Reply:
This doesn't work inside the table, we'd have to move it to after the table, if we want to use an admonition.

Reply:
It is possible. See https://github.com/asciidoctor/asciidoctor/issues/2113.

Reply:
Well, despite what that issue says, the latest antora-preview doesn't render it.

Reply:
I've moved the note below the table, imo that works better than having the note take away the focus from the rest of the table contents.